docs/topics/db/managers.txt

```diff
@@ -381,7 +381,7 @@ this base class::
             abstract = True
 
 If you use this directly in a subclass, ``objects`` will be the default
-manager if you declare no managers in the base class::
+manager if you declare no managers in the subclass::
```

Comment:
Nice find 👍  though I just wonder whether we should use "child class" to be consistent with the next example? 🤔

Reply:
@ebram96 Can you update your patch? :point_up:

Reply:
Sure, I'm updating it now.